definitions/checks/foreman/verify_login_session_count.rb

```diff
@@ -1,8 +1,9 @@
-class Checks::ResetLoginSessionIds < ForemanMaintain::Check
+class Checks::VerifyLoginSessionCount < ForemanMaintain::Check
```

Comment:
In addition to this, you will need to move your check class under `foreman` module.

```
module Checks
  module Foreman
      class VerifyLoginSessionCount < ForemanMaintain::Check
          ....
      end
  end
end
```